ansible/roles/uan_interfaces/tasks/can.yml

```diff
@@ -200,14 +214,54 @@
     debug:
       msg: "{{chn_device_list.stdout_lines }}"
 
+  - name: "Determine if ifcfg-{{ chn_device_list.stdout_lines | first }} exists"
+    stat:
+      path: "/etc/sysconfig/network/ifcfg-{{ chn_device_list.stdout_lines | first }}"
+    register: ifcfg_chn
+    
+  - name: "Add CHN IP to {{ chn_device_list.stdout_lines | first }}"
+    when: (can_ip is defined) and (not ifcfg_chn.stat.exists)
+    shell: "ip address add {{ can_ip }}/{{ can_prefix }} dev {{ chn_device_list.stdout_lines | first }}"
+    register: cmd_out
+    failed_when: cmd_out.rc not in [0, 1, 2]
+
+  - name: "Add static route for {{ customer_access_gateway }}"
+    when:
+      - can_ip is defined
+      - not ifcfg_chn.stat.exists
+    shell: "ip route add {{ customer_access_network }} dev {{ chn_device_list.stdout_lines | first }} src {{ can_ip }}"
+    register: cmd_out
+    failed_when: cmd_out.rc not in [0, 1, 2]
+
+  - name: "Add static route for {{ customer_access_gateway }} MetalLB services"
+    when:
+      - can_ip is defined
+      - not ifcfg_chn.stat.exists
+    shell: "ip route add {{ customer_access_network_metallb }} via {{ customer_access_gateway }} dev {{ chn_device_list.stdout_lines | first }}"
```

Comment:
Should this be `chn_device_list` here if this is for CAN rather than CHN?

Reply:
This is only operating for CHN mode (see lines 275-276):
```
  when: 
    - (uan_user_access_cfg | upper) == "CHN"
 ```
 
I didn't have a 1.2 CAN system available to test adding this route. Seeing as this wasn't noticed for CAN in the previous UAN releases I didn't add it for CAN routing.